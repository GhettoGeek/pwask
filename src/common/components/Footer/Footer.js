import React from 'react'
import { Grid } from '@material-ui/core'
import { makeStyles } from '@material-ui/core/styles'
import BottomNavigation from '../BottomNavigation'

const useStyles = makeStyles({
  root: {
    width: '100%',
    position: 'fixed',
    bottom: 0,
    left: 0,
  },
})

function Footer() {
  const classes = useStyles()

  return (
    <Grid
      item
      xs={12}
      id="footer"
      component="footer"
      className={classes.root}
    >
      <BottomNavigation />
    </Grid>
  )
}

export default Footer
