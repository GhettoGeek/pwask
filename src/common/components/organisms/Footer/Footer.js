import React from 'react'
import PropTypes from 'prop-types'
import { withTranslation } from 'react-i18next'
import { Grid, Typography } from '@material-ui/core'
import { styled } from '../../../utils/style'

const Wrapper = styled('footer')((theme) => ({
  marginTop: theme.spacing(1),
  padding: theme.spacing(1),
}))

const Footer = ({ t }) => (
  <Wrapper>
    <Grid container justify="space-evenly">
      <Typography variant="subtitle1" align="center" color="textSecondary" component="p">
        {t('footer.title')}
      </Typography>
    </Grid>
  </Wrapper>
)

Footer.propTypes = {
  t: PropTypes.func.isRequired,
}

export default withTranslation()(Footer)
