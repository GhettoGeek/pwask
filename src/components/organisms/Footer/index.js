import React from 'react'
import PropTypes from 'prop-types'
import { withTranslation } from 'react-i18next'
import Grid from '@material-ui/core/Grid'
import Typography from '@material-ui/core/Typography'
import { styled } from '../../../utils/style'

const Wrapper = styled('footer')(theme => ({
  marginTop: theme.spacing.s,
  padding: theme.spacing.s,
}))

const Footer = ({ t }) => (
  <Wrapper>
    <Grid container justify="space-evenly">
      <Typography variant="subtitle1" align="center" color="textSecondary" component="p">
        {t('footer.title')}
      </Typography>
    </Grid>
  </Wrapper>
)

Footer.propTypes = {
  t: PropTypes.func.isRequired,
}

export default withTranslation()(Footer)
