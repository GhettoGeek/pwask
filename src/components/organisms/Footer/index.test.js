import React from 'react'
import renderer from 'react-test-renderer'

import Footer from '.'

describe('Footer', () => {
  it('Should render the component as expected', () => {
    const component = renderer.create(<Footer />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
