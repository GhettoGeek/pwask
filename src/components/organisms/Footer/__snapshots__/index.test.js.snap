// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Should render the component as expected 1`] = `
<div>
  footer
</div>
`;
